optuna_dashboard/ts/apiClient.ts

```diff
@@ -30,6 +30,7 @@ interface TrialResponse {
   system_attrs: Attribute[]
   note: Note
   artifacts: Artifact[]
+  constraints: number[]
```

Comment:
[nits] If we want to align with the other fields such as `values?`, should it be also an optional field?